tinyagentos/restart_orchestrator.py

```
@@ -11,30 +12,56 @@
     )
 
 
 def read_pending_restart() -> dict | None:
-    if not PENDING_RESTART_PATH.exists():
+    path = _pending_restart_path()
+    if not path.exists():
         return None
     try:
-        return json.loads(PENDING_RESTART_PATH.read_text())
+        return json.loads(path.read_text())
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)